packages/connect-popup/src/static/popup.css

```diff
@@ -2370,8 +2370,8 @@ section > div .divider span:after {
 .select-device .wrapper .select-device-list button .trezor_icon.trezor_icon_t2t1 {
     background-image: url(./images/trezor-t2t1.png);
 }
-.select-device .wrapper .select-device-list button .trezor_icon.trezor_icon_t2b1 {
-    background-image: url(./images/trezor-t2b1.png);
+.select-device .wrapper .select-device-list button .trezor_icon.trezor_icon_t3b1 {
```

Comment:
This class name should stay, since T3B1 is already defined on L2376

Reply:
fff902b2cd5f1151632a1a3d0a78e3da791d7201